test/JDBC/expected/ownership_chain_procedure.out

```diff
@@ -0,0 +1,273 @@
+-- tsql
+create database db1;
+go
+
+use db1;
+go
+
+create login l1 with password='123';
```

Comment:
Please prefix all objects with the test file name, so that when something breaks we can easily locate the root cause

Reply:
Will address it after adding all the tests.